src/Recommendations/Command/GenerateDatabaseCommand.php

```diff
@@ -48,7 +54,7 @@ private function tableExists(string $table): bool
 
     private function createTables(bool $drop, Table ...$tables)
     {
-        foreach ($tables as $table) {
+        foreach (array_reverse($tables) as $table) {
```

Comment:
@giorgiosironi This solves the database creation/deletion issues. This way we delete the tables in the reverse order they are created